tests/test-mappings-post-taxonomy-term-rule-validator.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+use Wordlift\Mappings\Validators\Rule_Validator;
+use Wordlift\Mappings\Validators\Post_Taxonomy_Term_Rule_Validator;
+use Wordlift\Mappings\Mappings_REST_Controller;
+use Wordlift\Mappings\Jsonld_Converter;
+
+/**
+ * This test checks if the post taxonomy term validator is correct.
+ * @group mappings
+ */
+class Post_Taxonomy_Term_Rule_Validator_Test extends Wordlift_Unit_Test_Case {
+
+    /**
+     * Our expected route for rest api.
+     */
+    protected $mapping_route = '/wordlift/v1/mappings/get_taxonomy_terms';
+
+	/**
+	 * @var Taxonomy_Term_Rule_Validator
+	 */
+	private $instance;
+
+	public function setUp() {
+		$this->instance = new Post_Taxonomy_Term_Rule_Validator();
+
+        $this->rest_instance = new Mappings_REST_Controller();
+        $this->rest_instance->register_routes();
+        global $wp_rest_server, $wpdb;
+
+        $wp_rest_server = new WP_REST_Server();
+        $this->server   = $wp_rest_server;
+        $this->wpdb = $wpdb;
+        do_action( 'rest_api_init' );
+	}
+
+    /**
+     * Test is rest route exists for fetching post taxonomy & terms.
+     */
+    public function test_rest_route_for_fetching_post_taxonomy_terms() {
```

Comment:
This test is failing in a configuration, make sure test is compatible with minimum version 4.4